models/personel.py

```diff
@@ -0,0 +1,21 @@
+class Personel(object):
+    """personel model"""
+
+    def __init__(self, name):
+        self.name = name
+        want_accommodation = 'N'
+
+
+class staff(Personel):
```

Comment:
**Code Quality: Critical**

Always use upper case for your class names. See PEP-8.